Yank/yank.py

```diff
@@ -465,12 +465,13 @@ def expand_cutoff(passed_system, max_allowed_cutoff = 16 * unit.angstroms):
             noninteracting_expanded_state = None
         else:
             # Create the system for noninteracting
-            noninteracting_expanded_system = copy.deepcopy(alchemical_system)
+            expanded_factory = AbsoluteAlchemicalFactory(fully_interacting_expanded_state.system,
```

Comment:
This should expand the `CustomNonbondedForce` `rcutoff` and `alpha_ewald` variables, yes?